voip_utils/sip.py

```diff
@@ -48,6 +49,51 @@ def caller_rtcp_port(self) -> int:
         return self.caller_rtp_port + 1
 
 
+@dataclass
+class SipEndpoint:
+    """Information about a SIP endpoint."""
+
+    host: str
+    port: int
+    username: str
+    description: str
+
+    @property
+    def sip_uri(self) -> str:
+        """Return the URI for the SIP endpoint."""
+        if self.username is not None:
+            return f"sip:{self.username}@{self.host}:{self.port}"
+        else:
+            return f"sip:{self.host}:{self.port}"
+
+
+def get_rtp_port(body: str) -> Tuple[Optional[int], Optional[int]]:
+    body_lines = body.splitlines()
+    rtp_port = None
+    opus_payload_type = None
+    for line in body_lines:
+        line = line.strip()
+        if line:
+            key, value = line.split("=", maxsplit=1)
+            if key == "m":
+                parts = value.split()
+                if parts[0] == "audio":
+                    rtp_port = int(parts[1])
+                elif key == "a" and value.startswith("rtpmap:"):
+                    # a=rtpmap:123 opus/48000/2
```

Comment:
This is not possible, as you are inside the `key == "m"` block. Should this have 1 higher indentation?